app/Resources/views/committee/show.html.twig

```diff
@@ -4,6 +4,37 @@
 
 {% block banner '' %}
 
+{% block javascripts %}
+    <script type="text/javascript">
+        Kernel.onLoad(function(di) {
+            var page = 1;
+            var limit = {{ committee_timeline_max_messages }};
+            var moreButton = dom('#committee-timeline-more');
+
+            on(moreButton, 'click', function() {
+                page++;
+                di.get('api').getCommitteeTimelineFeed(
+                    '{{ committee.uuid }}',
+                    '{{ committee.slug }}',
+                    ((page - 1) * limit),
+                    function (response) {
+                        if (response instanceof XMLHttpRequest) {
+                            hide(moreButton);
+                            return;
+                        }
```

Comment:
When there are no more pages, the response is empty and the button must be removed. The strange thing is, my callback function doesn't receive an empty string but an XMLHttpRequest object.